### DISCUSSION TITLE
Is the RabbitMQ parameter modification configuration effective in real-time and does it require a reboot to take effect?

### DISCUSSION THREAD

**Discussion by charlienss** [rabbitmq/rabbitmq-server#9051] [Other]
Is the RabbitMQ parameter modification configuration effective in real-time and does it require a reboot to take effect?

**Comment by lukebakken** [comment]
It depends on the parameter. If you're talking about a configuration setting in `rabbitmq.conf`, then yes, a reboot is required.
